infrastructure/rdfdb/cdk/bin/deploy_to_ecr.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Set variables
+REPO_NAME="rdf4j"
+REGION="us-east-1"  # Replace with your desired region
+DOCKER_FILE_PATH="../../docker"  # Adjust this path as needed
+
+# Create ECR repository
+aws ecr create-repository --repository-name $REPO_NAME --region "us-east-1"
+
+# Get the repository URI
+REPO_URI=$(aws ecr describe-repositories --repository-names $REPO_NAME --region "us-east-1" --query 'repositories[0].repositoryUri' --output text)
```

Comment:
You have REGION defined above replace your hard coded version of it with the REGION variable.